ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -581,6 +585,44 @@ def test_ipa_healthcheck_no_errors(self):
         )
         assert returncode == 0
 
+    def test_ipa_healthcheck_no_errors_with_overrides(self):
+        """
+        Test overriding command-line options in a configuration file.
+        """
```

Comment:
You can add code to retrieve ipa-healthcheck version (we already have `tasks.get_healthcheck_version`) and call `pytest.skip` if the version is < 0.10.
```
        version = tasks.get_healthcheck_version(self.master)
        if parse_version(version) < parse_version("0.10"):
            pytest.skip("Skipping test for 0.10 healthcheck version")
```

Reply:
I will eventually.